### PR TITLE
Feature/render country info

### PR DESCRIPTION
**Pull request by callumpb**
Feature/render country info

### CODE REVIEW

File: client/src/views/country_info.js [hunk 1]
```diff
@@ -3,7 +3,16 @@ const CountryInfo = function () {
 }
 
 CountryInfo.prototype.render = function (country) {
+	console.log(country);
```
Comment (callumpb) [inline client/src/views/country_info.js:6]: remove this please

Comment (jaylonez) [reply client/src/views/country_info.js:6]: This line of code is integral to our codebase.

Comment (callumpb) [reply client/src/views/country_info.js:6]: better remove it or ill drop your database

Comment (jaylonez) [reply client/src/views/country_info.js:6]: It's too late. It's been committed. Enjoy having the country you selected logged to console every time you select it in the dropdown!

**Pull request merged**